packages/noya-state/src/layers.ts

```diff
@@ -10,6 +10,8 @@ import {
 
 export type ParentLayer = Extract<Sketch.AnyLayer, { layers: any }>;
 
+export type ChildLayer = Exclude<Sketch.AnyLayer, { layers: any }>;
```

Comment:
This seems like a useful type. Some parent layers can also be children (group and shapeGroup), so I might go with:

```ts
export type ChildLayer = Exclude<
  Sketch.AnyLayer,
  { _class: 'artboard' | 'symbolMaster' | 'page' }
>;
```